public/apps/accountinfo/accountinfo.html

```diff
@@ -0,0 +1,51 @@
+<div class="searchguard app-container ng-scope">
+
+    <div class="kuiLocalNav">
+        <div class="kuiLocalNavRow">
+            <div class="kuiLocalNavRow__section">
+                <div class="kuiLocalTitle">
+                    Account Information
+                    <br />
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12">
+
+                <div class="kuiHeaderBar" style="margin-bottom: 20px; margin-top: 30px;">
+                    <div class="kuiHeaderBarSection">
+                        <h2 class="kuiSubTitle">Username</h2>
+                    </div>
+                </div>
+                <ul>
+                    <li>{{ctrl.sg_user.user_name}}</li>
+                </ul>
+
+                <div class="kuiHeaderBar" style="margin-bottom: 20px; margin-top: 30px;">
+                    <div class="kuiHeaderBarSection">
+                        <h2 class="kuiSubTitle">Search Guard Roles</h2>
+                    </div>
+                </div>
+                <p data-ng-repeat="role in ctrl.sg_user.sg_roles" style="margin-bottom: 10px;">
+                    {{role}}
+                </p>
+                <div ng-show="ctrl.sg_user.sg_roles == 0" style="font-style: italic">No Search Guard roles found, please check the role mapping for this user.</div>
```

Comment:
Should this be 


```suggestion
                <div ng-show="ctrl.sg_user.sg_roles.length == 0" style="font-style: italic">No Search Guard roles found, please check the role mapping for this user.</div>
```
Should this be ctrl.sg_user.sg_roles.length?